controllers/add_services.go

```diff
@@ -56,6 +56,7 @@ func (a addServices) reconcile(ctx context.Context, r *FoundationDBClusterReconc
 			}
 			owner := internal.BuildOwnerReference(cluster.TypeMeta, cluster.ObjectMeta)
 			service.ObjectMeta.OwnerReferences = owner
+			log.Info("Creating service", "name", service.Name)
```

Comment:
That should be a debug statement not an info statement. I would prefer if we also log the corresponding namespace and cluster name.